src/ExpressiveInstaller/OptionalPackages.php

```diff
@@ -42,27 +42,27 @@ class OptionalPackages
     /**
      * @var array
      */
-    private static $config;
+    public static $config;
```

Comment:
Why public?

If it's for testing, the various `assertAttribute*()` assertions can do assertions on non-public properties. That said, I've not tried on static properties; if those present an issue, I'd argue reflection is still a better way to test these.